Tests/JobsTests/JobStorageTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  JobStorageTests.swift
+//  Async
+//
+//  Created by Raul Riera on 2019-03-16.
+//
+
+import XCTest
+@testable import Jobs
+
+final class JobStorageTests: XCTestCase {
+	func testStringRepresentationIsValidJSON() {
+		let jobStorage = JobStorage(key: "vapor", data: Data(), maxRetryCount: 1, id: "identifier", jobName: "jobs")
+		
+		XCTAssertEqual(jobStorage.stringValue(), "{\"data\":\"\",\"key\":\"vapor\",\"maxRetryCount\":1,\"id\":\"identifier\",\"jobName\":\"jobs\"}")
```

Comment:
I like the idea behind this test, but the `stringValue()` function is deterministic so we can't rely on the ordering being the same every time. Perhaps we can decode the string value back into a new struct and compare the properties?